### DISCUSSION TITLE
Quartz not Indexed

### DISCUSSION THREAD

**Discussion by biapontes** [mtex-toolbox/mtex#980] [Ask Anything]
Hello,

I'm Beatriz and I'm having some problems on indexing my **quartizite** samples. There are only two samples that are having the same indexing problem. My script:

%% Specify Crystal and Specimen Symmetries

% crystal symmetry
CS = {... 
  'notIndexed',...
 crystalSymmetry('-3m1', [4.913 4.913 5.504], 'X||a*', 'Y||b', 'Z||c*', 'mineral', 'Quartz-new', 'color', 'light red'),...
 crystalSymmetry('12/m1', [5.058 8.763 19.111], [90,95.39,90]*degree, 'X||a*', 'Y||b', 'Z||c', 'mineral', 'Mica', 'color', 'cyan')};

% plotting convention
setMTEXpref('xAxisDirection','east');
setMTEXpref('zAxisDirection','outOfPlane');

And I get the image below:

![notIndexed](https://user-images.githubusercontent.com/82956260/115571398-55e79800-a295-11eb-8cfe-d7e48fbb41cb.jpg)

I think is the crystal symmetry syntax, because the band contrast image is perfectly done.

<img width="414" alt="Map_BC_" src="https://user-images.githubusercontent.com/82956260/115570939-e671a880-a294-11eb-829b-50b4d42f21eb.png">

 But I don't know which symmetry to replace on the script. I think this is a simple error to solve... Could someone help me?

Thanks a lot!!

**Comment by kilir** [comment]
Hi,
your map looks like it was stitched. There are other reports here that apparently some of the softwares used for that produce an output with messed-up the phase IDs.
What is the output of `unique(ebsd.phase)` ?

Cheers,
Rüdiger
